sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.UnitTest/HttpParsers/DocumentDbHttpParserTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenTelemetry.Exporter.AzureMonitor.Models;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.AzureMonitor.HttpParsers
+{
+    public class DocumentDbHttpParserTests
+    {
+        [Fact]
+        public void DocumentDbHttpParserConvertsValidDependencies()
+        {
+            Dictionary<string, string> defaultProperties = null;
+            var databaseProperties
+                = new Dictionary<string, string> { ["Database"] = "myDatabase" };
+            var collectionProperties
+                = new Dictionary<string, string> { ["Database"] = "myDatabase", ["Collection"] = "myCollection" };
+            var sprocProperties
+                = new Dictionary<string, string> { ["Database"] = "myDatabase", ["Collection"] = "myCollection", ["Stored procedure"] = "mySproc" };
+            var udfProperties
+                = new Dictionary<string, string> { ["Database"] = "myDatabase", ["Collection"] = "myCollection", ["User defined function"] = "myUdf" };
+            var triggerProperties
+                = new Dictionary<string, string> { ["Database"] = "myDatabase", ["Collection"] = "myCollection", ["Trigger"] = "myTrigger" };
+
+            string defaultResultCode = "200";
+
+            var testCases = new List<Tuple<string, string, string, Dictionary<string, string>, string>>()
+            {
+                // Database operations
+                Tuple.Create("Create database",             "POST",     "https://myaccount.documents.azure.com/dbs", defaultProperties, defaultResultCode),
```

Comment:
@pakrym wish to get your perspective here.

I think we want the DocumentDB SDK to put information indicating the logical operation name, instead of coupling the logic inside the Azure Monitor Exporter.

Reply:
That's what we have been doing for track 2 SDKs but I'm not sure how much leeway we would have for existing libraries. We might need to keep these implementations for back-compat.

Reply:
Just to throw some thoughts here - would it be possible for us to only support track 2 SDKs in this exporter.
Folks who are using legacy SDKs can use ApplicationInsights SDK, and we'll continue to support it until X% of customers move to track 2 SDKs + OpenTelemetry .NET + Azure Monitor Exporter?

Reply:
Agree we shouldn't do any complex domain specific parsing in the exporter. Its good idea to callout that only track2 SDKs are supported in Otel + this exporter.

Reply:
> Agree we shouldn't do any complex domain specific parsing in the exporter. Its good idea to callout that only track2 SDKs are supported in Otel + this exporter.

Yeah, I think this exporter will only have knowledge about the Azure Monitor schema/endpoint/protocol. It should have no idea what Service Bus, Cosmos DB and Azure Search are.

Reply:
Removed all HTTP parsers. Now we check based on az.namespace tag